marshmallow/fields.py

```diff
@@ -360,6 +362,10 @@ def __init__(self, default=None, attribute=None, error=None,
         else:
             raise ValueError("The 'validate' parameter must be a callable "
                             'or a collection of callables.')
+
+        if not (isinstance(required, bool) or isinstance(required, basestring)):
```

Comment:
I'm not sure that this check is necessary. Given the implementation of `ValidationError`, the user could pass a `str`, `dict`, or `list` and the result would be valid.